python/py-luma.core/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
```

Comment:
are the tests not included on PyPI; if there is no obvious reason to use GitHub typically getting Python packages from PyPI (using the `python` PG) is preferred.

Reply:
The test data needed to successfully run the tests isn’t included in PyPI. Do you want me to add a comment to that effect?

Reply:
no thanks, that's a good enough reason and doesn't need an explanation in the Portfile.